cypress/test/logout.test.cy.js

```diff
@@ -0,0 +1,32 @@
+import 'cypress-testing-library/add-commands';
+import cy from 'cypress';
+
+import 'dotenv/config';
+import process from 'process-env';
+
+process.env.NODE_ENV = 'test';
```

Comment:
You shouldn't edit env variables, only read from them. This could be used to store login information, for example